bot/connector-web-model/src/main/kotlin/ai/tock/bot/connector/web/send/UrlButton.kt

```diff
@@ -22,5 +22,6 @@ import com.fasterxml.jackson.annotation.JsonTypeName
 data class UrlButton(
     val title: String,
     val url: String,
+    val openInSameWindow: Boolean = false,
```

Comment:
Maybe `target` would be a better name than `window` and `targetBlank` or `targetSelf` better, to fit W3C attribute? (https://www.w3schools.com/tags/att_a_target.asp)

Reply:
I changed the name of the attribute to target and added an enum that gathers the possible values of the target param to fit W3C values for the target attribute